lib/core/src/connection_pool.cpp

```diff
@@ -277,7 +280,12 @@ namespace irods
             return;
         }
 
-        if (clientLogin(ctx.conn.get()) != 0) {
+        RodsEnvironment env{};
+        if (const auto err = getRodsEnv(&env); err < 0) {
+            THROW(err, "Failed to get client environment.");
+        }
```

Comment:
You can actually replace these lines with `_getRodsEnv` since `getRodsEnv` never fails unless passed a nullptr.

Reply:
True. I'll replace the new uses with this. I was a little hesitant to use it given the leading underscore, but it really isn't much different from `getRodsEnv`.

Reply:
Done